docs/course_materials/exercises/01_Linux/class_exercise_3.md

```diff
@@ -0,0 +1,12 @@
+### Let's Practice Together \#3
+
+- Open a new file with the vim editor
+- Enter insert mode
+- Write 4 sentences about your self
```

Comment:
it should be 4 sentences in 4 different lines.
maybe we can tell them to paste a text, for example: 
add this text to the fie: 
"Roses are red,
Violets are blue,
Sugar is sweet,
And so are you.

Roses are red,
Violets are blue,
White wine costs less,
Than dinner for two.

Roses are red,
But violets aren’t blue,
They’re purple, you dope,
Now go get a clue.
"

and then you can  another question and ask then to search "Roses" , 
a bonus question is to replace all "Roses" with "watermelons"